gen/solid.hoon

```diff
@@ -11,26 +11,44 @@
   !:
 :-  %say
 |=  $:  {now/@da eny/@uvJ bec/beak}
-        {$~ $~}
+        {arg/$@($~ {top/path $~}) dub/_|}
     ==
 :-  %noun
-=+  top=`path`/(scot %p p.bec)/[q.bec]/(scot %da now)/sys
-=+  pax=`path`(weld top `path`[%hoon ~])
-=+  arp=`path`(weld top `path`[%ovra ~])
+?~  arg  $(arg ~[top=`path`/(scot %p p.bec)/[q.bec]/(scot %da now)/sys])
+=+  pax=`path`(weld top.arg `path`[%hoon ~])
+=+  arp=`path`(weld top.arg `path`[%ovra ~])
```

Comment:
This is a little weird. Wouldn't we want this to be `%arvo`, not `%ovra`? @cgyarvin